src/utilities/color/color.ts

```diff
@@ -88,3 +103,26 @@ export function getColorForTheme(color: string) {
     const requiredColor = color.substr(5).toUpperCase();
     return Colors[`${requiredColor}${AppStore.Instance.darkTheme ? "4" : "2"}`];
 }
+
+function generateColorGradientArray(targetColorHex: string, startColorHex = "#000000", steps: number = 1024) {
+    const gradientArray = [];
+
+    const targetColor = tinycolor(targetColorHex).toRgb();
+    const startColor = tinycolor(startColorHex).toRgb();
+
+    for (let i = 0; i <= steps - 1; i++) {
+        // Calculate the interpolation factor
+        const factor = i / (steps - 1);
+
+        // Interpolate RGBA values from the start color to the target color
+        const red = Math.round((1 - factor) * startColor.r + factor * targetColor.r);
+        const green = Math.round((1 - factor) * startColor.g + factor * targetColor.g);
+        const blue = Math.round((1 - factor) * startColor.b + factor * targetColor.b);
+        const alpha = 255;
+
+        // Push the RGBA values as a string to the array
```

Comment:
Do you mean we convert the numbers to strings?

Reply:
I forgot to modify the comment...